GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -3236,6 +3241,7 @@ private void HandleDrop(DragEventArgs e)
                 return;
             }
 
+            CommitInfoTabControl.SelectedTab = TreeTabPage;
```

Comment:
Similar to text drop, should check that the file is in this repo

Reply:
Right, simply move this line to the few lines below, just after ``if (fileTree.SelectFileOrFolder(path)) {`` and before the ``return``